tmeister/static/src/menu/main-menu.component.js

```diff
@@ -1,24 +1,62 @@
 import React from 'react';
-import styles from './main-menu.styles.css';
-import CprTooltip from 'cpr-tooltip';
+import { Tooltip, Paper } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+  menu: {
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    height: '100%',
+    width: '64px',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+  },
+  root: {
+    backgroundColor: theme.palette.primary.main,
+  },
+  menuItem: {
+    padding: 8,
+    textDecoration: 'none',
+    '&.active' : {
+      backgroundColor: theme.palette.primary.dark,
+    },
+    '&:hover': {
+      backgroundColor: theme.palette.primary.dark,
+    },
+  },
+  icon: {
+    width: '100%',
+    height: '100%'
+  }
+}))
 
 export default function MainMenu(props) {
-	return (
-		<div className={`${styles.menu}`}>
-			<CprTooltip text="Manage toggles">
-				<a className={`${styles.menuItem} ${activeIcon("toggles")}`} title="Manage toggles" href="/#/toggles">
-					<img src='/resources/toggle-icon.png' className={`${styles.icon}`} />
-				</a>
-			</CprTooltip>
-			<CprTooltip text="Audit trail">
-				<a className={`${styles.menuItem} ${activeIcon("audit-trail")}`} href="/#/audit-trail">
-					<img src='/resources/audit-icon.png' className={`${styles.icon}`} />
-				</a>
-			</CprTooltip>
-		</div>
-	);
+  const classes = useStyles()
+  return (
+    <Paper className={classes.menu} classes={classes}>
```

Comment:
NIT: passing `classes` seems a little weird 🤷‍♀️

Reply:
I was surprised by it as well: https://material-ui.com/api/paper/#props